pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1349,7 +1373,7 @@ func TestPreempt(t *testing.T) {
 				t.Errorf("unexpected error in preemption: %v", err)
 			}
 			if (node != nil && node.Name != test.expectedNode) || (node == nil && len(test.expectedNode) != 0) {
-				t.Errorf("expected node: %v, got: %v", test.expectedNode, node)
+				t.Errorf("expected node: %v, got: %v", test.expectedNode, node.GetName())
```

Comment:
Note for reviewers: logging out whole node object seems is too verbose.

Reply:
I agree.